frontend/src/pages/HomePage/index.js

```diff
@@ -8,26 +10,28 @@ import styles from "./index.module.css";
 
 const HomePage = () => {
   return (
-    <div>
-      <Header>Subvisual</Header>
-      <div>
-        <div className={styles.container}>
-          <Title>Welcome to Subvisual&apos;s Wheel of Fortune!</Title>
+    <>
+      <Header>{COMPANY}</Header>
+      <div className={styles.title}>
+        <Title>Welcome to {COMPANY}&apos;s Wheel of Fortune!</Title>
+      </div>
+
+      <div className={styles.container}>
+        <div className={styles.inner}>
```

Comment:
I don't think `inner` is very descriptive, what do you think about something like `callToAction`, `about` or `informationBlock`?